apps/passport-server/package.json

```diff
@@ -8,7 +8,7 @@
     "start:dev": "NODE_OPTIONS=--max_old_space_size=4096 NODE_ENV=development ts-node-dev --inspect=4321 --respawn --watch=.env,.env.local src/main.ts -p ./tsconfig.json",
     "dev:worker": "nodemon -w src/multiprocess/worker.ts --exec \"yarn tsc -p ./tsconfig.json\"",
     "dev": "yarn tsc -p ./tsconfig.json && concurrently -r \"yarn start:dev\" \"yarn dev:worker\"",
-    "start": "NODE_OPTIONS=--max_old_space_size=4096 NODE_ENV=production PORT=8080 node -r source-map-support/register build/src/main.js",
+    "start": "NODE_OPTIONS=${NODE_OPTIONS:---max_old_space_size=4096} NODE_ENV=production PORT=8080 node -r source-map-support/register build/src/main.js",
```

Comment:
do u have a link to docs that could explain what's going on here a bit? I'm not sure I'm able to parse this syntax.

Reply:
It's shell "parameter expansion": https://www.gnu.org/software/bash/manual/html_node/Shell-Parameter-Expansion.html

`${ENV_VAR:-DEFAULT}` will use `ENV_VAR` if set and not empty, and `DEFAULT` otherwise. The middle part is `:-`. So we want to use the environment's `NODE_OPTIONS` if it's set, or `--max_old_space_size=4096` if it's not, giving us `${NODE_OPTIONS:---max_old_space_size=4096}`. Then we can define `NODE_OPTIONS` to be whatever we'd prefer in the VM's environment variable configuration.